index.js

```diff
@@ -221,19 +221,26 @@ function runDataIntegrityProofFormatTests({
           proof.proofValue.should.be.a('string');
         }
       });
-    it('The "proof.proofValue" field MUST be a multibase-encoded ' +
-      'base58-btc encoded value.', function() {
-      this.test.cell = {columnId, rowId: this.test.title};
-      const multibase = 'z';
-      proofs.some(proof => {
-        const value = proof?.proofValue;
-        return value.startsWith(multibase) && shouldBeBs58(value);
-      }).should.equal(
-        true,
-        'Expected "proof.proofValue" to be multibase-encoded ' +
-          'base58-btc value.'
-      );
-    });
+    it('"proof.proofValue" MUST be a multibase-encoded encoded value.',
+      function() {
+        this.test.cell = {columnId, rowId: this.test.title};
+
+        const expectedPrefix = cryptosuite =>
+          cryptosuite === 'ecdsa-sd-2023' ? 'u' : 'z';
+        const isExpectedEncoding = ({cryptosuite, proofValue}) =>
+          cryptosuite === 'ecdsa-sd-2023' ?
+            shouldBeBase64Url(proofValue) : shouldBeBs58(proofValue);
+
+        proofs
+          .filter(proof => !!proof)
+          .some(({proofValue, cryptosuite}) => {
+            return proofValue.startsWith(expectedPrefix(cryptosuite)) &&
+              isExpectedEncoding({proofValue, cryptosuite});
+          }).should.equal(
+            true,
+            'Expected "proof.proofValue" to be multibase-encoded value'
+          );
+      });
```

Comment:
This simple assert doesn't expose _which_ expected format. I'd like to do that for clarity, but interestingly the original assertion is using "some", so it isn't clear how to do this. I can expose the expected encoding in the assertion message